src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: 1 200px;
```

you don't need this `1` value here
```suggestion
  grid-template-columns: 200px;
```